server/src/test-integration/java/com/thoughtworks/go/server/service/AgentConfigServiceIntegrationTest.java

```diff
@@ -1,737 +0,0 @@
-/*
- * Copyright 2019 ThoughtWorks, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.thoughtworks.go.server.service;
-
-import com.thoughtworks.go.config.*;
-import com.thoughtworks.go.config.exceptions.EntityType;
-import com.thoughtworks.go.domain.AgentInstance;
-import com.thoughtworks.go.helper.AgentInstanceMother;
-import com.thoughtworks.go.helper.AgentMother;
-import com.thoughtworks.go.i18n.LocalizedMessage;
-import com.thoughtworks.go.listener.AgentStatusChangeListener;
-import com.thoughtworks.go.server.domain.AgentInstances;
-import com.thoughtworks.go.server.domain.Username;
-import com.thoughtworks.go.server.service.EnvironmentConfigService;
-import com.thoughtworks.go.server.service.result.HttpLocalizedOperationResult;
-import com.thoughtworks.go.util.GoConfigFileHelper;
-import com.thoughtworks.go.util.ReflectionUtil;
-import com.thoughtworks.go.util.SystemEnvironment;
-import com.thoughtworks.go.util.TriState;
-import org.apache.commons.lang3.StringUtils;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.test.context.ContextConfiguration;
-import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
-
-import java.util.*;
-
-import static org.hamcrest.Matchers.*;
-import static org.hamcrest.Matchers.is;
-import static org.junit.Assert.*;
-
-@RunWith(SpringJUnit4ClassRunner.class)
-@ContextConfiguration(locations = {
-        "classpath:WEB-INF/applicationContext-global.xml",
-        "classpath:WEB-INF/applicationContext-dataLocalAccess.xml",
-        "classpath:testPropertyConfigurer.xml",
-        "classpath:WEB-INF/spring-all-servlet.xml",
-})
-public class AgentConfigServiceIntegrationTest {
-    @Autowired
```

Comment:
Can you please check if these removed tests are present in `AgentServiceIntegrationTest`? because we moved a few methods from `AgentConfigService` to `AgentService`

Reply:
Verified and added in [commit](https://github.com/gocd/gocd/pull/6271/commits/c1504122ec95bc326bddd0717b1965f7f2920675)